stackanetes/manifest.py

```diff
@@ -143,10 +144,16 @@ def render(self):
         return data
 
     def _find_template(self):
-        if self.type in GENERIC_TYPES:
-            return "generic-{}.yml.j2".format(self.type)
-        elif self.type == "fluentd-elasticsearch":
-            return "fluentd-elasticsearch.yml.j2"
+        if self.type.count('/') != 1:
+            msg = "Invalid format type of {} service. Service type should " \
+                  "have following format (generic|job)/(template_name)."
```

Comment:
Did you mean to use `custom` instead of `job` in `(generic|job)/(template_name)` ?